app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,9 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+<!--[if ( !IE ) | ( gte IE 9 ) ]><!-->
+<%# render 'dough/shared/social_sharing_icon_sprite' %>
```

Comment:
I'm not sure, I don't think they always had conditionals around them, I remember something breaking when I first moved them into the base but that was at the bottom, I think Ben suggested wrapping them in the comments. I guess we'd need to remove them and re-test on all browsers.

I've tried find the commit but my comments do not give a reason which is annoying.